lib/ip.go

```diff
@@ -145,16 +146,23 @@ func RandIP4Range(iprange IP4Range, noBogon bool) (net.IP, error) {
 	return net.IP(randIPbyte[:]), nil
 }
 
-// RandIP4ListWrite prints a list of new randomly generated IPv4 addresses.
+// RandIP4ListWrite prints a list of randomly generated IPv4 addresses.
 func RandIP4ListWrite(n int, noBogon bool) {
 	for i := 0; i < n; i++ {
 		fmt.Println(RandIP4(noBogon))
 	}
 }
 
-// RandIP4ListWrite prints a list of new randomly generated IPv4 addresses
-// within starting and IPs ending range.
-func RandIP4RangeListWrite(startIP, endIP string, n int, noBogon bool) error {
+// RandIP4ListWrite prints a list of randomly generated IPv4 addresses.
```

Comment:
Wrong function name: should be `RandIP4RangeListWrite`. Please check the other funcs to see if the same mistake was made or not.